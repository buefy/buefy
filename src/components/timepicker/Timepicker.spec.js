import { shallowMount } from '@vue/test-utils'
import BTimepicker from '@components/timepicker/Timepicker'

describe('BTimepicker', () => {
    it('is called', () => {
        const wrapper = shallowMount(BTimepicker)

        expect(wrapper.name()).toBe('BTimepicker')
        expect(wrapper.isVueInstance()).toBeTruthy()
    })
})
