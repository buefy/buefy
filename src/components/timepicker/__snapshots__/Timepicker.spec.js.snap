// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BTimepicker render correctly 1`] = `
<div class="timepicker control">
    <b-dropdown-stub maxheight="200" triggers="click" mobilemodal="true" animation="fade" trapfocus="true" closeonclick="true" canclose="true" appendtobodycopyparent="true">
        <b-dropdown-item-stub custom="true" focusable="true" ariarole="">
            <b-field-stub grouped="true" position="is-centered" addons="true">
                <b-select-stub usehtml5validation="true" statusicon="true" placeholder="00">
                    <option value="1">
                        1
                    </option>
                    <option value="2">
                        2
                    </option>
                    <option value="3">
                        3
                    </option>
                    <option value="4">
                        4
                    </option>
                    <option value="5">
                        5
                    </option>
                    <option value="6">
                        6
                    </option>
                    <option value="7">
                        7
                    </option>
                    <option value="8">
                        8
                    </option>
                    <option value="9">
                        9
                    </option>
                    <option value="10">
                        10
                    </option>
                    <option value="11">
                        11
                    </option>
                    <option value="0">
                        12
                    </option>
                </b-select-stub> <span class="control is-colon">:</span>
                <b-select-stub usehtml5validation="true" statusicon="true" placeholder="00">
                    <option value="0">
                        00
                    </option>
                    <option value="1">
                        01
                    </option>
                    <option value="2">
                        02
                    </option>
                    <option value="3">
                        03
                    </option>
                    <option value="4">
                        04
                    </option>
                    <option value="5">
                        05
                    </option>
                    <option value="6">
                        06
                    </option>
                    <option value="7">
                        07
                    </option>
                    <option value="8">
                        08
                    </option>
                    <option value="9">
                        09
                    </option>
                    <option value="10">
                        10
                    </option>
                    <option value="11">
                        11
                    </option>
                    <option value="12">
                        12
                    </option>
                    <option value="13">
                        13
                    </option>
                    <option value="14">
                        14
                    </option>
                    <option value="15">
                        15
                    </option>
                    <option value="16">
                        16
                    </option>
                    <option value="17">
                        17
                    </option>
                    <option value="18">
                        18
                    </option>
                    <option value="19">
                        19
                    </option>
                    <option value="20">
                        20
                    </option>
                    <option value="21">
                        21
                    </option>
                    <option value="22">
                        22
                    </option>
                    <option value="23">
                        23
                    </option>
                    <option value="24">
                        24
                    </option>
                    <option value="25">
                        25
                    </option>
                    <option value="26">
                        26
                    </option>
                    <option value="27">
                        27
                    </option>
                    <option value="28">
                        28
                    </option>
                    <option value="29">
                        29
                    </option>
                    <option value="30">
                        30
                    </option>
                    <option value="31">
                        31
                    </option>
                    <option value="32">
                        32
                    </option>
                    <option value="33">
                        33
                    </option>
                    <option value="34">
                        34
                    </option>
                    <option value="35">
                        35
                    </option>
                    <option value="36">
                        36
                    </option>
                    <option value="37">
                        37
                    </option>
                    <option value="38">
                        38
                    </option>
                    <option value="39">
                        39
                    </option>
                    <option value="40">
                        40
                    </option>
                    <option value="41">
                        41
                    </option>
                    <option value="42">
                        42
                    </option>
                    <option value="43">
                        43
                    </option>
                    <option value="44">
                        44
                    </option>
                    <option value="45">
                        45
                    </option>
                    <option value="46">
                        46
                    </option>
                    <option value="47">
                        47
                    </option>
                    <option value="48">
                        48
                    </option>
                    <option value="49">
                        49
                    </option>
                    <option value="50">
                        50
                    </option>
                    <option value="51">
                        51
                    </option>
                    <option value="52">
                        52
                    </option>
                    <option value="53">
                        53
                    </option>
                    <option value="54">
                        54
                    </option>
                    <option value="55">
                        55
                    </option>
                    <option value="56">
                        56
                    </option>
                    <option value="57">
                        57
                    </option>
                    <option value="58">
                        58
                    </option>
                    <option value="59">
                        59
                    </option>
                </b-select-stub>
                <!---->
                <b-select-stub usehtml5validation="true" statusicon="true" value="AM">
                    <option value="AM">
                        AM
                    </option>
                    <option value="PM">
                        PM
                    </option>
                </b-select-stub>
            </b-field-stub>
            <!---->
        </b-dropdown-item-stub>
    </b-dropdown-stub>
</div>
`;
